Lesson19/selcan/recursion.js

```diff
@@ -0,0 +1,142 @@
+// See completed in the lesson examples in the lesson files folder: https://github.com/frontend-from-0/group091123-files
+
+// 4. Write a recursive function to remove all occurrences of a specified character from a string.
+const sentenceEx5 = 'Hello, how are you?';
+const charToRemove = 'o';
+ function removeO(arr,val){
+	if(arr.length===0){
+		return;
```

Comment:
When using just a 'return', return value is undefined ('Hell, hw are yu?undefined'). Instead, return an empty string.
```suggestion
		return '';
```